_posts/2024-06-30-concurrent-search-follow-up.md

```diff
@@ -14,277 +14,294 @@ has_math: true
 has_science_table: true
 ---
 
-In October 2023, we [introduced concurrent segment search (CS) in OpenSearch](https://opensearch.org/blog/concurrent_segment_search/) as an experimental feature. CS improves search latency across a large variety of workloads. This feature was made generally available in OpenSearch 2.12 and we highly recommend that OpenSearch users try it. Here, we’ll share our exploration of performance in simulations of different real world scenarios. In particular, we’ll look at the performance trends as the available resources in the system decrease and concurrency increases.
+In October 2023, we [introduced concurrent segment search in OpenSearch](https://opensearch.org/blog/concurrent_segment_search/) as an experimental feature. Searching segments concurrently improves search latency across a large variety of workloads. This feature was made generally available in OpenSearch 2.12; we highly recommend that you try it! Here, we’ll share our exploration of performance in simulations of different real-world scenarios. In particular, we’ll look at the performance trends as the available resources in the system decrease and concurrency increases.
 
-In concurrent segment search, each shard-level search request on a node is divided into multiple execution tasks called slices. Slices can be executed concurrently on separate threads in the index_searcher threadpool, separate from the search threadpool. Each slice searches within the segments associated with it. Once all slice executions are complete, the collected results from all slices are combined (reduced) and returned to the coordinator node. The index_searcher threadpool is used to execute the slices of each shard search request and is shared across all shard search requests on a node. By default, the index_searcher threadpool has twice as many threads as the number of available processors.
+In concurrent segment search, each shard-level search request on a node is divided into multiple execution tasks called _slices_. Slices can be executed concurrently on separate threads in the index searcher thread pool, separate from the search thread pool. Each slice searches within the segments associated with it. Once all slice executions are complete, the collected results from all slices are combined (reduced) and returned to the coordinator node. The index searcher thread pool is used to execute the slices of each shard search request and is shared across all shard search requests on a node. By default, the index searcher thread pool has twice as many threads as the number of available processors.
```

Comment:
I meant to format it as `index_searcher` and `search` thread pools as those are the names of the threadpools.